### PR TITLE
[zh] adjust kubeadm_update_plan.md

### PR DESCRIPTION
**Pull request by my-git9**
[zh] adjust content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md

```
./scripts/lsync.sh content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md
content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md is still in sync

```

[zh] adjust content/zh/docs/tutorials/stateless-application/guestbook.md
```
./scripts/lsync.sh content/zh/docs/tutorials/stateless-application/guestbook.md
content/zh/docs/tutorials/stateless-application/guestbook.md is still in sync
```

### CODE REVIEW

**Comment by tengqm** [comment]
@my-git9 Can you combine changes like this into fewer PRs? This is a waste of gate resource and reviewer's time.

**Comment by my-git9** [comment]
> @my-git9 Can you combine changes like this into fewer PRs? This is a waste of gate resource and reviewer's time.

ok

File: content/zh/docs/tutorials/stateless-application/guestbook.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ min-kubernetes-server-version: v1.14
 <!--
 This tutorial shows you how to build and deploy a simple _(not production ready)_, multi-tier web application using Kubernetes and [Docker](https://www.docker.com/). This example consists of the following components:
 -->
-本教程向您展示如何使用 Kubernetes 和 [Docker](https://www.docker.com/) 构建和部署
+本教程向你展示如何使用 Kubernetes 和 [Docker](https://www.docker.com/) 构建和部署
 一个简单的 _(非面向生产的)_ 多层 web 应用程序。本例由以下组件组成：
```
Comment (tengqm) [inline content/zh/docs/tutorials/stateless-application/guestbook.md:32]:
```suggestion
本教程向你展示如何使用 Kubernetes 和 [Docker](https://www.docker.com/)
构建和部署一个简单的 **(非面向生产的)**多层 web 应用程序。本例由以下组件组成：
```

**Pull request merged**